tokamak.dev/Linter.md

```diff
@@ -0,0 +1,24 @@
+## Tokamak Linter
+
+Tokamak Linter is module to check your Tokamak projects for known warning
+and show them in xcode or in terminal.
+
+## How to run
+
+### Add to Xcode build phase
+
+- build TokamakCLI with Xcode
+- add path to TokamakCLI to build phase
```

Comment:
this should contain a snippet that could be added to a build phase, similar to what SwiftLint has in its README